cms/static/sass/elements/_tender-widget.scss

```diff
@@ -219,9 +219,9 @@
   display: block;
   width: 100%;
   height: auto !important;
-  -webkit-box-shadow: 0 1px 0 rgba(255,255,255,0.3) inset,0 0 0 rgba(0,0,0,0);
-  -moz-box-shadow: 0 1px 0 rgba(255,255,255,0.3) inset,0 0 0 rgba(0,0,0,0);
-  box-shadow: 0 1px 0 rgba(255,255,255,0.3) inset,0 0 0 rgba(0,0,0,0);
+  -webkit-box-shadow: 0 1px 0 rgba(255,255,255,0.3) inset,0 0 0 $transparent;
+  -moz-box-shadow: 0 1px 0 rgba(255,255,255,0.3) inset,0 0 0 $transparent;
+  box-shadow: 0 1px 0 rgba(255,255,255,0.3) inset,0 0 0 $transparent;
```

Comment:
hmm... I opted to punt this one to a future cleanup since a lot of this file uses deprecated/older prefixes we shouldn't need to worry about. I added it to the "NEXT" section of this PR so it is tracked somewhere.